apinotes/ModelIO.apinotes

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Please don't add them here. We're about to get rid of all the existing ones, and it's just a coincidence that they don't have any yet.